alacritty/src/display/mod.rs

```diff
@@ -367,6 +368,9 @@ pub struct Display {
     /// The ime on the given display.
     pub ime: Ime,
 
+    /// The state of the timer for frame scheduling.
+    pub frame_timer_info: FrameTimerInfo,
```

Comment:
```suggestion
    pub frame_timer: FrameTimer,
```

"Info" is a pretty useless suffix usually, I think we can just remove it.